### PR TITLE
[stable/openebs]: update openebs charts to 1.12.0 release

### PR DESCRIPTION
**Pull request by mittachaitu**
Signed-off-by: mittachaitu <sai.chaithanya@mayadata.io>

This PR update openebs charts to 1.12.0 release
- Updated README
- Updated Chart version
- Updated values.yaml

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/openebs]`)

### CODE REVIEW

File: charts/openebs/README.md [hunk 1]
```diff
@@ -59,31 +59,31 @@ The following table lists the configurable parameters of the OpenEBS chart and t
...
-| `snapshotOperator.provisioner.imageTag` | Image Tag for Snapshot Provisioner            | `1.11.0`                                  |
+| `snapshotOperator.provisioner.imageTag` | Image Tag for Snapshot Provisioner            | `1.12.0`                                  |
 | `snapshotOperator.controller.image`     | Image for Snapshot Controller                 | `openebs/snapshot-controller`             |
-| `snapshotOperator.controller.imageTag`  | Image Tag for Snapshot Controller             | `1.11.0`                                  |
+| `snapshotOperator.controller.imageTag`  | Image Tag for Snapshot Controller             | `1.12.0`                                  |
 | `snapshotOperator.replicas`             | Number of Snapshot Operator Replicas          | `1`                                       |
 | `ndm.enabled`                           | Enable Node Disk Manager                      | `true`                                    |
 | `ndm.image`                             | Image for Node Disk Manager                   | `openebs/node-disk-manager-amd64`         |
```
Comment (akhilerm) [inline charts/openebs/README.md:85]: NDM images have been made multiarch
```suggestion
| `ndm.image`                             | Image for Node Disk Manager                   | `openebs/node-disk-manager`         |
```

File: charts/openebs/README.md [hunk 2]
```diff
@@ -96,26 +96,26 @@ The following table lists the configurable parameters of the OpenEBS chart and t
 | `ndm.probes.enableSeachest`             | Enable Seachest probe for NDM                 | `false`                                   |
 | `ndmOperator.enabled`                   | Enable NDM Operator                           | `true`                                    |
 | `ndmOperator.image`                     | Image for NDM Operator                        | `openebs/node-disk-operator-amd64`        |
```
Comment (akhilerm) [inline charts/openebs/README.md:98]:
```suggestion
| `ndmOperator.image`                     | Image for NDM Operator                        | `openebs/node-disk-operator`        |
```

File: charts/openebs/values.yaml [hunk 3]
```diff
@@ -89,7 +89,7 @@ snapshotOperator:
 ndm:
   enabled: true
   image: "openebs/node-disk-manager-amd64"
```
Comment (akhilerm) [inline charts/openebs/values.yaml:91]:
```suggestion
  image: "openebs/node-disk-manager"
```

File: charts/openebs/values.yaml [hunk 4]
```diff
@@ -112,7 +112,7 @@ ndm:
 ndmOperator:
   enabled: true
   image: "openebs/node-disk-operator-amd64"
```
Comment (akhilerm) [inline charts/openebs/values.yaml:114]:
```suggestion
  image: "openebs/node-disk-operator"
```

File: charts/openebs/README.md (hunk 1 above)
Comment (mittachaitu) [reply charts/openebs/README.md]: done

File: charts/openebs/README.md (hunk 2 above)
Comment (mittachaitu) [reply charts/openebs/README.md]: Done

File: charts/openebs/values.yaml (hunk 4 above)
Comment (mittachaitu) [reply charts/openebs/values.yaml]: Done

**Pull request merged**